src/index.js

```diff
@@ -24,6 +24,11 @@ const isNestedArray = arr => bool(
 
 const isMap = m => bool(Object.prototype.toString.call(m) === '[object Map]')
 
+// check if consumer is yielding our effect to immediatly cause the generator function to throw an error
+const shouldThrowError = obj => bool(obj && Object.keys(obj).includes('@THROW'))
+
+const throwError = message => ({ '@THROW': message })
```

Comment:
My intent was just to make it clear that the magic string on this object is coming from the `redux-saga-test-engine` library and not any user code.

I don't think we need to copy the redux-saga object (at least not yet), I was displaying it for reference.